android/app/src/main/java/labs/pooh/eaterslab/repository/CafeteriasRepository.kt

```diff
@@ -69,6 +69,21 @@ class CafeteriasRepository(private val connectionStatusNotifier: ConnectionStatu
         return if (currentMenuOptionData.isNotEmpty()) currentMenuOptionData else null
     }
 
+    suspend fun getMenuOptionsOfCafeteria(cafeteriaId: Int): List<FixedMenuOptionDao>? {
```

Comment:
I think this should never be a null, empty list at most.

Reply:
nullability for fingle item is easierto get checked in Kotlin (using the ? operator you write data?.let { use(it) } instead of writing if (data.isNotEmpty()) use(data))